tests/scripts/thread-cert/node.py

```diff
@@ -1250,6 +1264,21 @@ def remove_prefix(self, prefix):
         self.send_command(cmd)
         self._expect('Done')
 
+    def get_prefixes(self):
+        netdata = self.netdata_show()
+        prefixes = []
+
+        for i in range(1, len(netdata)):
+            if netdata[i].startswith("Routes:"):
+                break
+            prefixes.append(netdata[1])
+
+        return prefixes
+
+    def netdata_show(self):
```

Comment:
implement `get_network_data` which returns well defined network data as dict?

Reply:
let's leave it for future PR (I already have the change, but would like to submit it in separate PR).